x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/create.ts

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { getTestAlertData } from './utils';
+import { UserAtSpaceScenarios } from '../../scenarios';
+import { getUrlPrefix } from '../../../common/lib/space_test_utils';
+import { FtrProviderContext } from '../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function createAlertTests({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+
+  describe('create', () => {
+    const createdObjects: Array<{ spaceId: string; id: string; type: string }> = [];
+
+    after(async () => {
+      await Promise.all(
+        createdObjects.map(({ spaceId, id, type }) => {
+          return supertest
+            .delete(`${getUrlPrefix(spaceId)}/api/${type}/${id}`)
+            .set('kbn-xsrf', 'foo')
+            .expect(204);
+        })
+      );
+    });
+
+    async function getScheduledTask(id: string) {
+      return await es.get({
+        id: `task:${id}`,
+        index: '.kibana_task_manager',
+      });
+    }
+
+    for (const scenario of UserAtSpaceScenarios) {
+      const { user, space } = scenario;
+      describe(scenario.id, () => {
+        it('create an alert', async () => {
```

Comment:
Here and elsewhere the description strings seem off.  Maybe something like `should handle create request appropriately`.